### DISCUSSION TITLE
Thoughts on rewire plugin / tooling

### DISCUSSION THREAD

**Discussion by glomotion** [swc-project/swc#2609] [Plugin request / idea]
To date, we have been rewiring JS imports (inside @cypress/react test suites etc) using a babel plugin inside our test build toolchain: https://github.com/asapach/babel-plugin-rewire-exports

Now that nextjs has shifted to using SWC - we will need to begin thinking about how we might handle module rewiring inside SWC based builds. 🤔 

Does anyone else have any thoughts about this? 
Has anyone managed to achieve similar functionality using SWC?

**Comment by pvcresin** [comment]
I want to replace my old repository using [babel-plugin-rewire](https://github.com/speedskater/babel-plugin-rewire) with SWC. Is there any good place to migrate to?
